components/playback/controls/player/DesktopPlayerControls.vue

```diff
@@ -359,6 +373,12 @@ export default {
         msg_one: "Data analysis is in progress for the recording snapshot. This won't take long.",
         msg_two: "Do not close the Mantarray software or power off the Mantarray instrument.",
       },
+      recording_snapshot_error_labels: {
+        header: "Error!",
+        msg_one: "An error occurred while running data analysis for the recording snapshot.",
+        msg_two: "Unable to converge due to bad quality of data.",
```

Comment:
I think we should reword this a little, maybe something like "Unable to process recording due to low quality calibration and/or noise", but double check with Hamed